samples/cpp/tutorial_code/HighGUI/AddingImagesTrackbar.cpp

```diff
@@ -27,11 +27,20 @@ Mat dst;
  * @function on_trackbar
  * @brief Callback for trackbar
  */
-static void on_trackbar( int, void* )
+static void on_trackbar( int pos , void* userdata )
```

Comment:
as a user i did not like and did not understand changes in this tutorial code. also i think if these changes need to be made related documentation files should be  updated. @berak @crackwitz what is your opinion.